extensions/wikia/Recirculation/js/helpers/LiftigniterHelper.js

```diff
@@ -34,6 +34,7 @@ define('ext.wikia.recirculation.helpers.liftigniter', [
 
 			// Callback renders and injects results into the placeholder.
 			w.$p('register', registerOptions);
+			w.$p("setRequestFields", ["rank", "thumbnail", "title", "url", "presented_by"]);
```

Comment:
Ok, I added `author` in 55c5a4d. I'm ok removing the fields as a supported configuration from the options. That's one less thing the user of this undocumented service needs to worry about.